requirements.json

```diff
@@ -61,6 +61,19 @@
     "sha256sum": "812c135ce00b570e7240e1d4fbc0b4e2a51928e97589ae9cf7aa08b0156ef6b4",
     "version": "10.4.0.GA"
   },
+  "jbosseap": {
+    "name": "Jboss Enterprise Application Platform",
+    "description": "",
+    "vendor": "Red Hat, Inc.",
+    "modulePath": "model/jbosseap",
+    "bundle": "no",
+    "targetFolderName": "jbosseap",
+    "dmUrl": "https://developers.redhat.com/download-manager/jdf/file/jboss-eap-7.0.0-installer.jar?workflow=direct",
+    "url": "",
```

Comment:
Just use 
http://download-node-02.eng.bos.redhat.com/released/JBEAP-7/7.0.0/jboss-eap-7.0.0-installer.jar
for url